capstone/src/main/webapp/group.html

```diff
@@ -8,8 +8,42 @@
       rel="stylesheet"
       type="text/css"
     />
+    <link rel="stylesheet" href="styles/group.css"></link>
+    <script src="scripts/poll.js"></script>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Group</title>
   </head>
-  <body></body>
-</html>
+
+  <body onload="getPollOptions()">
+    <div class="poll-container">
+      <div class="inner-poll-container">
+          <div class="section-text">
+              Poll
+          </div>
+          <div id="options-container"></div>
+          </div>
+        </br>
+        </br>
+        </br>
```

Comment:
What are you trying to achieve with these 3 line breaks? Could we perhaps achieve this using margin/padding?

Reply:
Yes, I just fixed it to add margin instead, thanks!